README.md

```diff
@@ -136,6 +136,7 @@ COMMANDS
   new       creates a new AsyncAPI file
   start     starts a new local instance of Studio
   validate  validate an AsyncAPI file
+  bundle    bundle multiple asyncapi files together
```

Comment:
sorry forgot about this one in last review

bundler is not only for bundling multiple asyncapi files, it can also bundle all references for a single document, right?

```suggestion
  bundle    bundle one or multiple asyncapi documents and their references together
```